tests/pipeline/test_example.py

```diff
@@ -17,7 +17,7 @@ def test_atm_fraud(self, snapshot: SnapshotTest):
                 "--pipeline-base-dir",
                 "examples",
                 "--defaults",
-                "./examples/bakdata/atm-fraud-detection/defaults",
+                "./examples/bakdata/atm-fraud-detection",
```

Comment:
is this argument still needed? it's also set in the config.yaml

Reply:
It's not, I will remove it.